txscript/data/script_tests.json

```diff
@@ -42,21 +42,117 @@
+["DATA_56 0x7a{56}", "'z'{56} EQUAL", "NONE", "OK", "Push 56 bytes"],
+["DATA_57 0x7a{57}", "'z'{57} EQUAL", "NONE", "OK", "Push 57 bytes"],
+["DATA_58 0x7a{58}", "'z'{58} EQUAL", "NONE", "OK", "Push 58 bytes"],
+["DATA_59 0x7a{59}", "'z'{59} EQUAL", "NONE", "OK", "Push 59 bytes"],
+["DATA_60 0x7a{60}", "'z'{60} EQUAL", "NONE", "OK", "Push 60 bytes"],
+["DATA_61 0x7a{61}", "'z'{61} EQUAL", "NONE", "OK", "Push 61 bytes"],
+["DATA_62 0x7a{62}", "'z'{62} EQUAL", "NONE", "OK", "Push 62 bytes"],
+["DATA_63 0x7a{63}", "'z'{63} EQUAL", "NONE", "OK", "Push 63 bytes"],
+["DATA_64 0x7a{64}", "'z'{64} EQUAL", "NONE", "OK", "Push 64 bytes"],
+["DATA_65 0x7a{65}", "'z'{65} EQUAL", "NONE", "OK", "Push 65 bytes"],
+["DATA_66 0x7a{66}", "'z'{66} EQUAL", "NONE", "OK", "Push 66 bytes"],
+["DATA_67 0x7a{67}", "'z'{67} EQUAL", "NONE", "OK", "Push 67 bytes"],
+["DATA_68 0x7a{68}", "'z'{68} EQUAL", "NONE", "OK", "Push 68 bytes"],
+["DATA_69 0x7a{69}", "'z'{69} EQUAL", "NONE", "OK", "Push 69 bytes"],
+["DATA_70 0x7a{70}", "'z'{70} EQUAL", "NONE", "OK", "Push 70 bytes"],
+["DATA_71 0x7a{71}", "'z'{71} EQUAL", "NONE", "OK", "Push 71 bytes"],
+["DATA_72 0x7a{72}", "'z'{72} EQUAL", "NONE", "OK", "Push 72 bytes"],
+["DATA_73 0x7a{73}", "'z'{73} EQUAL", "NONE", "OK", "Push 73 bytes"],
+["DATA_74 0x7a{74}", "'z'{74} EQUAL", "NONE", "OK", "Push 74 bytes"],
+["DATA_75 0x7a{75}", "'z'{75} EQUAL", "NONE", "OK", "Push 75 bytes"],
 
 ["NOTE: OP_PUSHDATA4 is not possible to use without error due to the minimal"],
 ["data requirements enforcing a smallest data push with it of 65,536 which is"],
 ["greater than the max script size of 16,384"],
 
+["OP_PUSHDATA1 0x4c 0x61{76}","'a'{76} EQUAL", "NONE", "OK", "Push 76 bytes"],
+["OP_PUSHDATA1 0x01 0x61{1}","'a'{1} EQUAL", "NONE", "ERR_MINIMAL_DATA", "OP_PUSHDATA1 instead of OP_DATA_1"],
+["OP_PUSHDATA2 0x0001 0x62{256}","'b'{256} EQUAL", "NONE", "OK", "Push 256 bytes"],
+["OP_PUSHDATA2 0x0100 0x62{256}","'b'{1} EQUAL", "NONE", "ERR_MINIMAL_DATA", "OP_PUSHDATA1 instead of OP_DATA_1"],
 ["0x4c 0x4c 0x61{76}","'a'{76} EQUAL", "NONE", "OK", "0x4c is OP_PUSHDATA1"],
 ["0x4d 0x0001 0x62{256}","'b'{256} EQUAL", "NONE", "OK", "0x4d is OP_PUSHDATA2"],
 ["0x4c01","0x01 NOP", "NONE", "ERR_MALFORMED_PUSH", "PUSHDATA1 with not enough bytes (stradle scripts to ensure data pushes are self-contained)"],
 ["0x4d0200ff","0x01 NOP", "NONE", "ERR_MALFORMED_PUSH", "PUSHDATA2 with not enough bytes (stradle scripts to ensure data pushes are self-contained)"],
 ["0x4e03000000ffff","0x01 NOP", "NONE", "ERR_MALFORMED_PUSH", "PUSHDATA4 with not enough bytes (stradle scripts to ensure data pushes are self-contained)"],
 
-["Cat related test coverage"],
+["1NEGATE", "-1 EQUAL", "NONE",  "OK", "Push -1 to the stack"],
+["RESERVED", "RESERVED EQUAL", "NONE", "ERR_RESERVED_OPCODE", "RESERVED is a reserved opcode"], 
+["TRUE", "1 EQUAL", "NONE",  "OK", "Push 1 to the stack"],
+["2", "2 EQUAL", "NONE",  "OK", "Push 2 to the stack"],
```

Comment:
This isn't doing what I suspect you probably think it is.  You'll need to specify `OP_2` or `0x52` so it isn't interpreted as a number.